server/server.js

```diff
@@ -136,11 +145,28 @@ var upload = multer({
  */
 var exonFile = bindingPath + 'exons.bin';
 
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
 /**
- * POST request is not used as it conflicts with jsonp.
+ * User info POST handler.
```

Comment:
This description doesn't match the handler content. It actually handles user-log as well.